cmd/minishift/cmd/delete.go

```diff
@@ -22,6 +22,7 @@ import (
 	"github.com/docker/machine/libmachine"
 	"github.com/minishift/minishift/pkg/minikube/cluster"
 	"github.com/minishift/minishift/pkg/minikube/constants"
+	instanceConfig "github.com/minishift/minishift/pkg/minishift/config"
```

Comment:
maybe _instanceState_ is better? _instanceState.Config_ seems to be more telling than _instanceConfig.Config_. WDYT?